src/inlines/Context.php

```diff
@@ -31,6 +31,18 @@ class Context {
   ];
 
   protected bool $isHtmlEnabled = false;
+  protected bool $isAllURIEnabled = false;
+  private keyset<string> $allowedURIs = keyset[];
+
+  public function enableAllURISchemes_UNSAFE(): this {
+    $this->isAllURIEnabled = true;
+    return $this;
+  }
+
+  public function enableURISchemeAllowlist(keyset<string> $allowlist): this {
```

Comment:
Calling this method when in isAllURIEnabled mode has no effect, since this list is ignored in that mode.
This would create an illusion of safety.
How could this API be improved?

Reply:
this should automatically disable 'all uri' mode